addon/components/one-way-select.js

```diff
@@ -165,7 +165,8 @@ const OneWaySelectComponent = Component.extend(DynamicAttributeBindings, {
       foundOption = options.find(findOption);
     }
 
-    if (optionTargetPath) {
+    if (optionTargetPath &&
+      foundOption !== undefined) {
```

Comment:
I think i got foundOption confused, so everything I wrote above can be ignored.

Technically it can only be undefined or what's found inside the options.
But `!isNone` is probably the best check because it's only true for [`undefined` or `null`](https://github.com/emberjs/ember.js/blob/v2.7.0/packages/ember-metal/lib/is_none.js#L1) which ember get also [isn't allowing as first get parameter](https://github.com/emberjs/ember.js/blob/v2.7.0/packages/ember-metal/lib/property_get.js#L48)